### PR TITLE
Update pretagging and object level docs

### PR DESCRIPTION
**Pull request by philippmwirth**
# Update pretagging and object level docs

- Clean up the docs a bit
- Show how to run with `pretagging=True` and `object_level.task_name=pretagging_lightly`
- Add docs on padding for object level

### CODE REVIEW

File: docs/source/docker/advanced/object_level.rst [hunk 1]
```diff
@@ -158,6 +164,39 @@ from python code. Examples on how to schedule the job are provided below.
...
+
+Padding
+-------
+Lightly makes it possible to add a padding around your bounding boxes. This allows
+for better visualization of the cropped images in the web-app and can improve the
+embeddings of the objects as the embedding model sees the objects in context. To add
+padding, simply specify `object_level.padding=X` where `X` is the padding relative
+to the bounding box size. For example, a padding of `X=0.1` will extend both width and
+height of all bounding boxes by 10 percent.
+
```
Comment (guarin) [inline docs/source/docker/advanced/object_level.rst:198]: Out of curiosity, is it possible to add a fixed width padding?

File: docs/source/docker/advanced/pretagging.rst [hunk 2]
```diff
@@ -85,12 +85,9 @@ Pretagging can be activated by passing the following argument to your docker
 run command: `pretagging=True`
 
 - `pretagging=True` enables the use of the pretagging model
-- `pretagging_debug=True` add a few images to the report for debugging showing
-  showing the image with the bounding box predictions. 
+- `pretagging_debug=True` add a few images to the report for debugging showing showing the image with the bounding box predictions.
```
Comment (guarin) [inline docs/source/docker/advanced/pretagging.rst:88]: Double `showing`

File: docs/source/docker/configuration/configuration.rst [hunk 3]
```diff
@@ -160,6 +160,8 @@ The following are parameters which can be passed to the container:
     # Name of the additional crop dataset on the Lightly Platform. A new dataset
     # is created if no dataset with this name exists.
     crop_dataset_name: ''
+    # Padding relative to the bbox size
+    padding: 0.0
```
Comment (guarin) [inline docs/source/docker/configuration/configuration.rst:164]: Should there not also be a `pretagging_upload` option?

File: docs/source/docker/advanced/code_examples/python_run_object_level_pretagging.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
+import lightly
+
+# Create the Lightly client to connect to the API.
+client = lightly.api.ApiWorkflowClient(token="TOKEN", dataset_id="DATASET_ID")
+
+# Schedule the docker run with the "object_level.task_name" argument set. 
+# All other settings are default values and we show them so you can easily edit
+# the values according to your need.
```
Comment (guarin) [inline docs/source/docker/advanced/code_examples/python_run_object_level_pretagging.py:8]: `pretagging` is not `True` by default.

**Review by guarin — COMMENTED** [review]
Looks good, just some minor things :)

File: docs/source/docker/advanced/object_level.rst (hunk 1 above)
Comment (philippmwirth) [reply docs/source/docker/advanced/object_level.rst:198]: Nope

**Review by guarin — APPROVED** [review]

**Pull request merged**